src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -304,60 +326,91 @@ internal static async Task CheckForUpdates()
         private static async Task<Release> QueryNewReleaseAsync(SemanticVersion baselineVersion)
         {
             bool isStableRelease = string.IsNullOrEmpty(PSVersionInfo.PSCurrentVersion.PreReleaseLabel);
-            string queryUri = isStableRelease ? LatestReleaseUri : Last3ReleasesUri;
+            string[] queryUris = s_notificationType switch
+            {
+                NotificationType.LTS => new[] { LTSBuildInfoURL },
```

Comment:
The default behavior is described in the comments of the enum definition:
- the preview version 'pwsh' checks for the new preview version and the new GA version.
- the GA version 'pwsh' checks for the new GA version only.

I don't expect regular users to configure this setting, so by default, it should be a bit smart and do the most-likely right thing based on the current pwsh version.
`lts` is made available, so that for users that only care about LTS release, they won't be bothered by the non-lts stable release.